### PR TITLE
#222 Gravatar user image has fixed size.

### LINKED ISSUE #222: On mobile, main menu avatar is stretched

### PR DESCRIPTION
**Pull request by criske**
fixes #222 
gravatar

### CODE REVIEW

**Comment by charlesmike** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by criske** [comment]
no bounty needed

**Comment by charlesmike** [comment]
@amihaiemil please review this Pull Request. Deadline (when it should be merged or closed) is ``2020-12-10T11:25:49.734595``.

You should check if the requirements have been implemented (partially or in full), if there are unit tests covering the changes and if the CI build passes. Feel free to reject the PR or ask for changes if it's too big or not clear enough.

Estimation here is ``30 minutes``, that's how much you will be paid. You will be paid even if this PR gets rejected.

**Comment by charlesmike** [comment]
@amihaiemil Don't forget to close this ticket before the deadline (2020-12-10T11:25:49). You are past the first half of the allowed period.

File: src/main/resources/public/css/main.css [hunk 1]
```diff
@@ -58,16 +58,11 @@ a.btn-outline-primary:hover {
...
-    width: 100%;
-    border: 1px solid #ddd;
-    box-shadow: 1px 1px 3px #ccc;
-}
-
 .gravatar_header img {
     padding: 3px;
-    width: 100%;
+    width: 48px;
+    width: 48px;
```
Comment (amihaiemil) [inline src/main/resources/public/css/main.css:64]: @criske You wrote ``width`` twice here. Did you want to say ``width`` and ``height``?

**Review by amihaiemil — CHANGES_REQUESTED** [review]
@criske one question :D

File: src/main/resources/public/css/main.css (hunk 1 above)
Comment (criske) [reply src/main/resources/public/css/main.css:64]: @amihaiemil yes, is  `height` :D

**Comment by amihaiemil** [comment]
@rultor merge it please

**Comment by charlesmike** [comment]
@amihaiemil thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).

**Pull request merged**